float64 s_scale  
float64 x_upperPosition 
float64 x_lowerPosition 
float64 x_upperVelocity
float64 x_lowerVelocity
float64 x_upperAccelera
float64 x_lowerAccelera
float64 x_0
float64 x_n
float64 x_v0
float64 x_vn
float64 y_upperPosition 
float64 y_lowerPosition 
float64 y_upperVelocity
float64 y_lowerVelocity
float64 y_upperAccelera
float64 y_lowerAccelera
float64 y_0
float64 y_n
float64 y_v0
float64 y_vn
float64 z_upperPosition 
float64 z_lowerPosition 
float64 z_upperVelocity
float64 z_lowerVelocity
float64 z_upperAccelera
float64 z_lowerAccelera
float64 z_0
float64 z_n
float64 z_v0
float64 z_vn
---
bool resultstatus
